packages/tc-ui/src/primitives/relationship/lib/relationship-picker.jsx

```diff
@@ -78,7 +79,7 @@ class RelationshipPicker extends React.Component {
 		// this is needed to prevent the event propagating up and then
 		// immediately clicking another button. VERY odd behaviour, and
 		// don't fully understand why, but this is the fix
-		event.preventDefault();
+		event.stopPropagation();
```

Comment:
ah - was this caused by the `label` thing too?

Reply:
not sure, I this is a more appropriate than preventDefault for this scenario

Reply:
Have you manually tested that the bad behaviour described in the comment doesn't happen after this change? If so, then :+1: